web/src/pages/projectInsights/DetailsTable.tsx

```diff
@@ -0,0 +1,903 @@
+        const rows = document.querySelectorAll('.ui.table tbody tr')
+        const rowData = Array.from(rows).map((row) => {
+            const cells = row.querySelectorAll('td')
+            return Array.from(cells).map((cell) => {
+                if (cell.textContent === '✅') {
+                    return 'TRUE'
+                } else if (cell.textContent === '❌') {
+                    return 'FALSE'
+                } else if (cell.querySelector('a')) {
+                    const anchor = cell.querySelector('a')
+                    return anchor ? anchor.href : null
+                } else {
+                    return cell.textContent
+                }
+            })
+        })
+
+        const separator = format === 'csv' ? ',' : '\t'
+        const fileData = [headerData, ...rowData].map((row) => row.join(separator)).join('\n')
+        const blob = new Blob([fileData], { type: `text/${format}` })
+        const url = URL.createObjectURL(blob)
+        const link = document.createElement('a')
+        link.href = url
+        // Generate the file name with the current date timestamp
+        const currentDate = new Date().toISOString().slice(0, 19)
+        const fileName = `project_insights_details_${currentDate}.${format}`
+        link.setAttribute('download', fileName)
+        document.body.appendChild(link)
+        link.click()
+        document.body.removeChild(link)
+    }
+
+    return (
+        <div>
+            <div style={{ textAlign: 'right' }}>
+                <Button onClick={() => exportToFile('csv')}>Export to CSV</Button>
+                <Button onClick={() => exportToFile('tsv')}>Export to TSV</Button>
+            </div>
+            <Table sortable>
+                <Table.Header>
+                    <Table.Row>
+                        <Table.HeaderCell
+                            className="header-cell"
+                            sorted={
+                                sortColumns.find((column) => column.column === 'dataset')?.direction
+                            }
+                            onClick={(event: React.MouseEvent<HTMLElement>) =>
+                                handleSort('dataset', event.shiftKey)
+                            }
+                        >
+                            Dataset
+                        </Table.HeaderCell>
+                        <Table.HeaderCell
+                            className="header-cell"
+                            sorted={
+                                sortColumns.find((column) => column.column === 'sequencing_type')
+                                    ?.direction
+                            }
+                            onClick={(event: React.MouseEvent<HTMLElement>) =>
+                                handleSort('sequencing_type', event.shiftKey)
+                            }
+                        >
+                            Seq Type
+                        </Table.HeaderCell>
+                        <Table.HeaderCell
```

Comment:
1. Pull this out as it's own component
2. You have a clear, what's required to actually this header cell
3. `[{name: "platform", title: "Platform", selectedOptions: selectedSeqPlatforms}, ...]`